Makefile

```diff
@@ -75,13 +75,14 @@ libbpfgo-static: $(VMLINUXH) | $(LIBBPF_OBJ)
 		.
 
 libbpfgo-static-test: libbpfgo-test-bpf-static
+	sudo env PATH=$(PATH) \
 	CC=$(CLANG) \
-		CGO_CFLAGS=$(CGO_CFLAGS_STATIC) \
-		CGO_LDFLAGS=$(CGO_LDFLAGS_STATIC) \
-		GOOS=linux GOARCH=$(ARCH) \
-		sudo -E -- go test \
-		-tags netgo -ldflags $(CGO_EXTLDFLAGS_STATIC) \
-		.
+	CGO_CFLAGS=$(CGO_CFLAGS_STATIC) \
```

Comment:
could you, pls, keep the indentation here as it was ? I'm going to "fix" the Makefile style soon, not only for this line but for other lines as well.

Reply:
Ack, no problem!